CodeStandards.md

```diff
@@ -0,0 +1,139 @@
+# ServUO Coding Standards
+
+## General Conventions
+
+### Use tabs for indents. 

```

Comment:
I think it would be reasonable to base off the `.editorconfig` that's present at the moment - including updating it and the ReSharper settings file that also in the repo with what (if anything) we end up deciding here.